src/scripts/main.js

```diff
@@ -1 +1,72 @@
 'use strict';
+
+const body = document.querySelector('body');
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+function success(result) {
```

Comment:
```suggestion
function handleSuccess(result) {
```
Let's get used to name function according to these rules:
![image](https://user-images.githubusercontent.com/96409563/211206243-e23e29e7-864e-4022-8754-2ef577ac1e9e.png)